js/src/analysis/classicEditorData.js

```diff
@@ -100,18 +103,25 @@ class ClassicEditorData {
 	subscribeToElements() {
 		this.subscribeToInputElement( "title", "title" );
 		this.subscribeToInputElement( "excerpt", "excerpt" );
+		this.subscribeToInputElement( "excerpt", "excerpt", false );
 		this.subscribeToInputElement( "excerpt", "excerpt_only" );
+		let cb = () => {
```

Comment:
I'd suggest to use a more meaningful variable name than `cb`, which is a bit too much WordPress-ish 😉